src/components/AvailableParkingSlotsView.tsx

```diff
@@ -41,15 +49,42 @@ export const AvailableParkingSlotsView: React.FC<IAvailableSlotsViewProps> = ({
     );
   }
 
+  function handleBuildingChange(
+    selectedBuilding: number,
+    totalBuildings: number
+  ): void {
+    setSelectedBuildingIndex(selectedBuilding);
+    setSelectedFloorIndex(0);
+  }
+
+  function handleFloorChange(selectedFloor: number, totalFloors: number): void {
```

Comment:
Make use of arrow function.

Reply:
Changed the function to arrow function not only that handleBuildingChange also changed